src/actions/tests/__snapshots__/ast.spec.js.snap

```diff
@@ -52,84 +52,4 @@ Array [
 ]
 `;
 
-exports[`ast setSymbols when the source is loaded should be able to set symbols 1`] = `
```

Comment:
erf, this is because of your config I think. Could you revert this file and src/actions/tests/__snapshots__/project-text-search.spec.js.snap from this PR ?

Reply:
Hooray, the circle ci passed. Do you know what the problem was before? And how did reverting those files change it?